generators/server/templates/gradle/swagger.gradle.ejs

```diff
@@ -36,6 +36,15 @@ openApiGenerate {
     importMappings = [Problem:"org.springframework.http.ProblemDetail"]
 }
 
+configurations {
+    all {
+        resolutionStrategy {
+            // TODO drop forced version. Refer to https://github.com/OpenAPITools/openapi-generator/issues/15876
+            force "io.swagger.parser.v3:swagger-parser:2.1.15"
```

Comment:
https://stackoverflow.com/questions/76288749/gradle-built-of-micronaut-based-app-breaks-with-nosuchmethoderror-from-3rd-party

Maybe we need to use 1.33?

Reply:
snakeyaml 2.0 is a major version.
Probably incompatible.
At link you provided says it’s not enough.

v2 is dependency of jackson 2.15.x